src/backend/distributed/relay/relay_event_utility.c

```diff
@@ -63,6 +63,13 @@ RelayEventExtendNames(Node *parseTree, uint64 shardId)
 		{
 			AlterSeqStmt *alterSeqStmt = (AlterSeqStmt *) parseTree;
 			char **sequenceName = &(alterSeqStmt->sequence->relname);
+			char **sequenceSchemaName = &(alterSeqStmt->sequence->schemaname);
```

Comment:
We create sequenceSchemaName, but never use it?

We talked about this, but do you know if we the code path hits here? It seems we don't support distributed sequences, still, we can ask the team for historical notes on this code path.